beets/autotag/mb.py

```diff
@@ -595,8 +595,12 @@ def album_info(release: Dict) -> beets.autotag.hooks.AlbumInfo:
 
     # Media (format).
     if release["medium-list"]:
-        first_medium = release["medium-list"][0]
```

Comment:
Is there a reason for tagging the whole thing with the media type of the first media?

Reply:
Want to recommend a gitsha I should look at?

Reply:
I think I found what you meant by the old UI and confirmed that it displayed bad info too. Here's what it looks like at 821e6296ab9516b09f0cad996a663c9115fad11e:

```
me@compy:/Volumes/Music/ti/hh/Four$ beet-test imp .

/Volumes/Music/ti/hh/Four (14 items)
Tagging:
    Hampton Hawes with Barney Kessel, Shelly Manne & Red Mitchell - Four!
URL:
    https://musicbrainz.org/release/03f118d7-99bb-4daa-ad58-ab86d5325398
Disambiguation string key albumrelease does not exist.
(Similarity: 97.7%) (media, year) (MusicBrainz, 2xHybrid SACD (CD layer), 2023, US, Contemporary Records, CR00501, Contemporary Records Acoustic Sounds Series)
Hybrid SACD (CD layer) 1
 * Yardbird Suite (#2-1)                  -> Yardbird Suite (#1-1)
 * There Will Never Be Another You (#2-2) -> There Will Never Be Another You (#1-2)
 * Bow Jest (#2-3)                        -> Bow Jest (#1-3)
 * Sweet Sue (#2-4)                       -> Sweet Sue (#1-4)
 * Up Blues (#2-5)                        -> Up Blues (#1-5)
 * Like Someone in Love (#2-6)            -> Like Someone in Love (#1-6)
 * Love Is Just Around the Corner (#2-7)  -> Love Is Just Around the Corner (#1-7)
Hybrid SACD (CD layer) 2
 * Yardbird Suite (#1-1)                  -> Yardbird Suite (#2-1)
 * There Will Never Be Another You (#1-2) -> There Will Never Be Another You (#2-2)
 * Bow Jest (#1-3)                        -> Bow Jest (#2-3)
 * Sweet Sue (#1-4)                       -> Sweet Sue (#2-4)
 * Up Blues (#1-5)                        -> Up Blues (#2-5)
 * Like Someone in Love (#1-6)            -> Like Someone in Love (#2-6)
 * Love Is Just Around the Corner (#1-7)  -> Love Is Just Around the Corner (#2-7)
[A]pply, More candidates, Skip, Use as-is, as Tracks, Group albums,
Enter search, enter Id, aBort, eDit, edit Candidates?
```

Reply:
Sorry for being very slow in responding these days. I was about to suggest looking into what was before the "new" UI was introduced in this PR: https://github.com/beetbox/beets/pull/3721

Seems like you found what you are looking for already, but I think the sha you posted is missing a character in the end? That was the merge commit, I think it was this one 821e6296